# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/moz.build
# Commit: da37e2d8a175
# Full Hash: da37e2d8a175fda08a98920d820dda0899b6aedf
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 15b92f98f62c -r da37e2d8a175 dom/credentialmanagement/identity/moz.build
--- a/dom/credentialmanagement/identity/moz.build	Thu Jun 12 18:16:52 2025 +0000
+++ b/dom/credentialmanagement/identity/moz.build	Thu Jun 12 18:16:53 2025 +0000
@@ -13,16 +13,22 @@
     "IdentityCredentialSerializationHelpers.h",
     "IdentityNetworkHelpers.h",
     "NavigatorLogin.h",
+    "WebIdentityChild.h",
+    "WebIdentityHandler.h",
+    "WebIdentityParent.h",
 ]
 
 IPDL_SOURCES += [
-    "IPCIdentityCredential.ipdlh",
+    "PWebIdentity.ipdl",
 ]
 
 UNIFIED_SOURCES += [
     "IdentityCredential.cpp",
     "IdentityNetworkHelpers.cpp",
     "NavigatorLogin.cpp",
+    "WebIdentityChild.cpp",
+    "WebIdentityHandler.cpp",
+    "WebIdentityParent.cpp",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")